doc/article/technical_article.tex

```diff
@@ -165,66 +161,61 @@ \subsection{Main Features}
 
 \section{Architecture}
 The trckr project consists of two separate applications. The business and data
-layer are  handled by the backend application, while the presentation layer is
-implemented through the frontend application. The backend provides a RESTful
-API, which is used by the frontend application to read and write the necessary
+layers are handled by the back-end application while the presentation layer is
+implemented through the front-end application. The back-end provides a RESTful
+API which is used by the front-end application to read and write the necessary
 data.
 
 This separation of concerns, achieved through the clear decoupling between the
-frontend and backend, allows for easy extensibility and fast development cycles.
-Additionally one backend is able to serve multiple client applications, without
-any further work required.
+front-end and back-end, allows for easy extensibility and fast development
+cycles. Additionally, a single back-end is able to serve multiple client
+applications without any further work required.
+
+\begin{figure}[h]
+    \includegraphics[width=8cm]{architecture}
+    \caption{Architecture of trckr}
+    \label{fig:architecture}
+\end{figure}
 
 \section{Technologies}
 \subsection{Django}
-Django is an open source web framework written in Python.\cite{django} It
+Django is an open-source web framework written in Python.\cite{django} It
 encourages fast, clean and simple development of web applications. One of the
 main advantages is its fast setup, enabling the developer to create applications
-swiftly through it's Model-View-Presenter scheme. Django also comes with support
-for various databases. Many users compare Django to Ruby On Rails but written in
-Python. Django also follows the DRY principle (Don't Repeat Yourself).
+swiftly through its Model-View-Presenter scheme. Django also comes with support
+for various databases and follows the DRY principle (Don't Repeat Yourself).
```

Comment:
usually when an abbreveation is used for the first time in a text it is written out and then the abbreveation in parentheses. because we do not use this abbreveation anymore I'd just put the written out in parentheses after the abbreveation.
"... follows the DRY (Don't Repeat Yourself) principle." **OR** "... follows the Don't Repeat Yourself (DRY) principle."
I'd say it's up to us which to use.